src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -6826,7 +6826,7 @@ public static void Main()
             var cmd = CreateCSharpCompiler(null, dir.Path, new[] { "/nologo", "a.cs", "/keycontainer:bbb", });
             var comp = cmd.CreateCompilation(TextWriter.Null, new TouchedFileLogger(), NullErrorLogger.Instance);
 
-            Assert.Equal(comp.Options.StrongNameProvider.GetType(), typeof(DesktopStrongNameProvider));
+            Assert.Equal(typeof(DesktopStrongNameProvider), comp.Options.StrongNameProvider.GetType());
```

Comment:
Should this use `Assert.IsType<T>(...)` instead? It looks like there are other instances where we're using `Assert.Equal(typeof(T), expr.GetType())`.

Reply:
Yes it should. I think this slipped through because I implemented the rules in the opposite order: fixed `typeof`, then expected vs. actual. Let me see if this registers after a rebuild.

---
In reply to: [327206264](https://github.com/dotnet/roslyn/pull/38789#discussion_r327206264) [](ancestors = 327206264)

Reply:
Strangely that gives no warning on rebuild. 

---
In reply to: [327314572](https://github.com/dotnet/roslyn/pull/38789#discussion_r327314572) [](ancestors = 327314572,327206264)

Reply:
Ah it's because the rule only checks for uses of `Assert.IsType` not of `Assert.Equal`. Odd distinction to make but it's one the tool makes today.

---
In reply to: [327315805](https://github.com/dotnet/roslyn/pull/38789#discussion_r327315805) [](ancestors = 327315805,327314572,327206264)